api/RTCRtpStreamStats.json

```diff
@@ -0,0 +1,730 @@
+{
+  "api": {
+    "RTCRtpStreamStats": {
```

Comment:
`RTCRTPStreamStats` was added in bug 902003 for Firefox 27: https://hg.mozilla.org/mozilla-central/rev/5372aea57cdb